doc-changelog/action.yml

```
@@ -68,7 +68,7 @@ inputs:
     required: false
     type: string
 
-  toml-version:
+  toml-version: # Deprecated input, remove in v11
```

```suggestion
  # TODO: Remove deprecated input in v11
```
Could you also mention that in the current logged error ? Same for other actions

---

One quick note that I removed `toml-version` from the generated inputs table in [985c7ae](https://github.com/ansys/actions/pull/828/commits/985c7ae89cd0e3e6c332d17641f6056e4bb2fc70). This is [how the table renders now](https://actions.docs.ansys.com/pull/828/style-actions/index.html#doc-style-action). I guess that's ok?